captum/insights/frontend/package.json

```diff
@@ -2,6 +2,7 @@
   "name": "frontend",
   "version": "0.2.0",
   "private": true,
+  "homepage": ".",
```

Comment:
Was this necessary to get relative paths working?

Reply:
Yup, this rewrites the urls from `/something.css` to `./something.css`.